src/tersets.zig

```diff
@@ -128,13 +128,17 @@ pub fn isWithinErrorBound(
     error_bound: f32,
 ) bool {
     if (uncompressed_values.len != decompressed_values.len) {
+        std.debug.print("\nHere not the same size\n", .{});
```

Comment:
Maybe remove the leftover debugging code.

Reply:
Ups, did not see this. My bad!